### PR TITLE
Fix DestinationRanges update when IP changes

### PR DESCRIPTION
**Pull request by sugangli**
L4 firewall's DestinationRanges was not being updated when the forwardingrule's IP was updated.

### CODE REVIEW

File: pkg/firewalls/firewalls_l4.go [hunk 1]
```diff
@@ -122,6 +122,10 @@ func firewallRuleEqual(a, b *compute.Firewall, skipDescription bool) bool {
 		}
 	}
 
+	if !utils.EqualStringSets(a.DestinationRanges, b.DestinationRanges) {
```
Comment (swetharepakula) [inline pkg/firewalls/firewalls_l4.go:125]: Can you add a test for this too?

**Review by swetharepakula — COMMENTED** [review]
Thanks Sugang for the bug fix!

Can you add a corresponding unit test for the change. Otherwise LGTM

File: pkg/firewalls/firewalls_l4.go (hunk 1 above)
Comment (sugangli) [reply pkg/firewalls/firewalls_l4.go:125]: Updated. PTAL.

**Review by swetharepakula — COMMENTED** [review]
Thanks for the change!

/lgtm
/approve

**Pull request merged**